src/assets/img/Arrows.tsx

```diff
@@ -0,0 +1,11 @@
+import { activeColors } from "../../styledConstants"
+
+type ArrowProps = {
+    isClicked?: boolean;
+}
+
+export const DownArrow = ({ isClicked }: ArrowProps) => {
+    return (<div className="svgWrap" style={isClicked ? { transform: 'rotate(180deg)' } : { transform: 'rotate(0deg)' }}><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
```

Comment:
Inline styles should be avoided if possible. I added the ability to add svg images without using a large piece of code, in the assets/icons folder you can add all icons (including those with changed color) and just import.

Reply:
I preffer not to use svg in img tag, cause svg is a tag itself, better use it as React Component. On my second commercial project every svg is a React Component. About inline styles I know, I just kinda forgot to fix it...